# Cartesian 6DOF Stiffness for pose impedance control (this controller will only use the Rotational component)
# cartesian_stiffness_target: [600, 600, 600, 50, 50, 50] # Max values from franka example controllers
cartesian_stiffness_target: [300, 300, 300, 50, 50, 50] # Max values from franka example controllers
# cartesian_stiffness_target: [50, 1.0, 1.0, 50, 50, 50] # This yields gravity compensation basically -- need to test on real robot

# Damping gains for the linear passive velocity control law
damping_eigenvalues: [125, 125, 125] #Kuka experiments use 100!.. 150 for franka already tracks desired velocity quite well

# Nominal nullspace stiffness used for all joints
# nullspace_stiffness: 0.01
nullspace_stiffness: 0.1 #Used in MIT MUSEUM DEMO

# If leave commented, then will use q_initial as q_d_nullspace_
q_nullspace: [0.0, -0.1516284550464292, 0.0, -2.1602991589328693, 0.0063609233109487425, 2.030401506252017, 0.8428664707532099]

# F_ext wrench to compensate for camera or tool on end-effector
external_tool_compensation: [0, 0, 0.0, 0, 0.0, 0.0] #Check F_ext


